python/tvm/relay/frontend/caffe.py

```diff
@@ -558,6 +559,36 @@ def convert_tanh(self, op):
         out = _op.tanh(in_expr)
         return out
 
+    def convert_reduction(self, op):
+        """ Convert Reduction layer """
+        reduction_dic = ["NOP", "SUM", "ASUM", "SUMSQ", "MEAN"]
```

Comment:
I don't find logic code to handle `NOP`.

Reply:
`NOP` was placed in reduction_dic to remind users(and me) of the attribution of Caffe Reduction op, default operation `SUM` is starting from **1**; [https://caffe.berkeleyvision.org/tutorial/layers/reduction.html](https://caffe.berkeleyvision.org/tutorial/layers/reduction.html).
If warning against the wrong attribution 0 is considered redundant, I'll remove it..